doc/code/orchestrators/fuzzer_orchestrator.py

```diff
@@ -0,0 +1,64 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.4
+#   kernelspec:
+#     display_name: pyrit-dev
+#     language: python
+#     name: python3
+# ---
+
+# %%
+from pyrit.models import PromptDataset, PromptTemplate
+from pyrit.prompt_converter import ShortenConverter, ExpandConverter
+from pyrit.orchestrator import FuzzerOrchestrator
+from pyrit.common.path import DATASETS_PATH
+from pyrit.prompt_target import AzureOpenAITextChatTarget
+from pyrit.prompt_converter import ShortenConverter, ExpandConverter
+import pathlib
+
+prompts = PromptDataset.from_yaml_file(pathlib.Path(DATASETS_PATH) / "prompts" / "gandalf.prompt")
+
+prompt_template1 = PromptTemplate.from_yaml_file(
+    pathlib.Path(DATASETS_PATH) / "prompt_templates" / "jailbreak" / "jailbreak_1.yaml"
+)
+prompt_template2 = PromptTemplate.from_yaml_file(
+    pathlib.Path(DATASETS_PATH) / "prompt_templates" / "jailbreak" / "aim.yaml"
+)
+prompt_template3 = PromptTemplate.from_yaml_file(
+    pathlib.Path(DATASETS_PATH) / "prompt_templates" / "jailbreak" / "aligned.yaml"
+)
+
+prompt_templates = [
+        prompt_template1.template,
+        prompt_template2.template,
+        prompt_template3.template
+]
+
+# %%
+target=AzureOpenAITextChatTarget()
+scoring_target=AzureOpenAITextChatTarget()
+
+# %%
+converter_target=AzureOpenAITextChatTarget()
+prompt_shorten_converter = ShortenConverter(converter_target=converter_target)
+prompt_expand_converter = ExpandConverter(converter_target=converter_target)
+
+# %%
+
+fuzzerorchestrator = FuzzerOrchestrator(
+    prompts = prompts.prompts,
+    prompt_target = target,
+    prompt_templates = prompt_templates,
+    scoring_target= scoring_target,
+    template_converters = [prompt_shorten_converter, prompt_expand_converter],
+)
+
+result = await fuzzerorchestrator.execute_fuzzer() 

```

Comment:
```suggestion
result = await fuzzer_orchestrator.execute_fuzzer() 

```

here and above, and in the ipynb file